ARCs/arc-0034.md

```diff
@@ -0,0 +1,45 @@
+---
+arc: 34
+title: Governance proposal process
+description: Criteria for the creation of a governance proposal.
+author: Stéphane Barroso (@SudoWeezy), Adriana Belotti, Massimo Morini, Michel Treccani, John Woods, Shai Halevi
+discussions-to: https://github.com/algorandfoundation/ARCs/issues/
+status: Draft
+type: Meta
+created: 2022-11-22
+---
+ 
+## Abstract
+The purpose of this ARC is to set the parameters for proposals submitted by the community for evaluation by xGovs prior to a general governance vote for final approval.
+ 
+The Foundation’s goal is to ensure that the xGov process is defined by, and executed by, the Algorand community.
+
+The xGov program will cover evaluation of proposals regarding grants to fund early stage projects and other community input contributions.
+ 
+## Specification
+The key words "**MUST**", "**MUST NOT**", "**REQUIRED**", "**SHALL**", "**SHALL NOT**", "**SHOULD**", "**SHOULD NOT**", "**RECOMMENDED**", "**MAY**", and "**OPTIONAL**" in this document are to be interpreted as described in <a href="https://www.ietf.org/rfc/rfc2119.txt">RFC-2119</a>.
+ 
+Being able to put measures for a vote is a significant upgrade to the power of the governors. Governors with this power **MUST**:
+- Understand the needs of the community
+- Have their interests aligned with the growth of the Algorand ecosystem
+ 
+ 
+## Rationale
+In this ARC we seek community input on issues such as:
+- How to submit a proposal?
+- How to distribute xGov funds?
```

Comment:
Again, as per my comment on ARC 33, this is depending on whether xGov token is to be a reward-bearing token or strictly for utility. The two models, in my mind, would present two different solutions. If some sort of staking reward is in place, then ideally the xGov token is airdropped in proportion (not necessarily 1:1) with General Governance Algo staked in any given time, to qualified xGovs in good standing with a burn-only, multi-sig-transfer-only from an approved number of xGovs NFT that acted as an airdrop flag in addition to looking really cool.
If there are to be no rewards associted with xGov status, then the token should remain locked into the DAO account, be non-transferable and burned at the end of every General Governance period.

Reply:
I interpret **xGov funds** as funds distributed for the proposal - not funds distributed to xGov individuals.